spec/models/suspended_email_spec.rb

```diff
@@ -62,7 +63,8 @@
           email_address: blocked_email,
         )
 
-        expect(SuspendedEmail.find_with_email_digest(digested_base_email)).to eq(blocked_email)
+        email_address = SuspendedEmail.find_with_email_digest(digested_base_email)&.email_address
+        expect(email_address).to eq(blocked_email)
```

Comment:
same as before, I'd update the spec to reflect the new return value of this method
```ruby
      it 'returns the SuspendedEmail' do
        blocked_email = FactoryBot.create(:email_address, email: 'blocked@example.com')
        digested_base_email = Digest::SHA256.hexdigest('blocked@example.com')
        suspended_email = FactoryBot.create(
          :suspended_email,
          digested_base_email: digested_base_email,
          email_address: blocked_email,
        )

        expect(SuspendedEmail.find_with_email_digest(digested_base_email)).to eq(suspended_email)
      end
```